core/src/main/java/org/owasp/encoder/Encode.java

```diff
@@ -243,7 +243,7 @@ public static void forHtmlContent(Writer out, String input)
      *
      * <b>Example JSP Usage</b>
      * <pre>
-     *     &lt;div&gt;&lt;%=Encode.forHtmlAttribute(unsafeData)%&gt;&lt;/div&gt;
+     *     &lt;input value=&quot;&lt;%=Encode.forHtml(unsafeData)%&gt;&quot; title=&#39;&lt;%=Encode.forHtml(moreUnsafeData)%&gt;&#39; /&gt;
```

Comment:
why would we use `forHtml` when outputing into an HTML Attribute?
```suggestion
     *     &lt;input value=&quot;&lt;%=Encode. forHtmlAttribute(unsafeData)%&gt;&quot; title=&#39;&lt;%=Encode. forHtmlAttribute(moreUnsafeData)%&gt;&#39; /&gt;
```